tools/bin/record_obfuscator.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+import hashlib
+import json
+import sys
+from typing import Any
+
+
+def _generate_hash(value: Any) -> str:
+    return hashlib.sha256(str(value).encode()).hexdigest()
+
+
+def obfuscate(value: Any) -> Any:
+    if isinstance(value, str):
+        obfuscated_value = "string_" + _generate_hash(value)
+    elif isinstance(value, (int, float)):
+        obfuscated_value = "number_" + _generate_hash(value)
+    elif isinstance(value, bool):
+        obfuscated_value = "boolean_" + _generate_hash(value)
+    elif value is None:
+        obfuscated_value = "null_" + _generate_hash(value)
+    elif isinstance(value, list):
+        obfuscated_value = "array_" + _generate_hash(json.dumps(value, sort_keys=True).encode())
```

Comment:
I think it'd also be helpful to add the length of string, numbers and array. Wdyt?